vizro-core/tests/e2e/vizro/dashboards/default/dashboard.py

```diff
@@ -1,4 +1,9 @@
 import e2e.vizro.constants as cnst
+from pages.actions_properties_shortcut_page import (
+    actions_default_property_controls_page,
+    ag_grid_underlying_id_shortcuts_page,
+    figures_title_header_footer_page,
```

Comment:
This is the last thing I would change. Everything else looks just perfect!! ⭐ 🚀 
```suggestion
    action_model_field_shortcut_page,
    action_ag_grid_underlying_id_shortcut_page,
    action_control_shortcut_page,
```

You just merged the PR in the moment I was writing the comment 😅 Would you mind adding this change in another PR you're working on 😃